app/models/user.rb

```diff
@@ -4,4 +4,8 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :confirmable, :lockable, :timeoutable, :trackable# and :omniauthable
 
+  def admin?
+    true # TODO - connect this to pundit
+  end
```

Comment:
this will need some Pundit implementation, but making anyone who logs in an admin atm. Is this ok w y'all?